lib/cli/kit/config.rb

```diff
@@ -0,0 +1,159 @@
+require 'fileutils'
+
+module CLI
+  module Kit
+    class Config
```

Comment:
Quick follow up here is to make `CLI::Kit::Config` subclass our IniParser (or use it)

Reply:
Yup. I have bad timing but there wasn't much transformation here so it's an easy follow-up